### PR TITLE
Add LED to Examples, update README

### PR DESCRIPTION
**Pull request by brentru**
Adding a LED blink to signify that a packet is sending/sent to The Things Network.

Updated README to incl. license information and correct PyPi installation instructions.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -26,6 +26,31 @@ Please ensure all dependencies are available on the CircuitPython filesystem.
 This is easily achieved by downloading
 `the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
 
+Installing from PyPI
+--------------------
+
+On supported GNU/Linux systems like the Raspberry Pi, you can install the driver locally `from
+PyPI <https://pypi.org/project/adafruit-circuitpython-bme280/>`_. To install for current user:
```
Comment (kattni) [inline README.rst]: Please replace `bme280` with `tinylora`.

File: README.rst [hunk 2]
```diff
@@ -26,6 +26,31 @@ Please ensure all dependencies are available on the CircuitPython filesystem.
...
+
+.. code-block:: shell
+
+    pip3 install adafruit-circuitpython-bme280
```
Comment (kattni) [inline README.rst]: Same as above.

File: README.rst [hunk 3]
```diff
@@ -26,6 +26,31 @@ Please ensure all dependencies are available on the CircuitPython filesystem.
...
+
+To install system-wide (this may be required in some cases):
+
+.. code-block:: shell
+
+    sudo pip3 install adafruit-circuitpython-bme280
```
Comment (kattni) [inline README.rst]: Same as above.

File: README.rst [hunk 4]
```diff
@@ -26,6 +26,31 @@ Please ensure all dependencies are available on the CircuitPython filesystem.
...
+
+To install in a virtual environment in your current project:
+
+.. code-block:: shell
+
+    mkdir project-name && cd project-name
+    python3 -m venv .env
+    source .env/bin/activate
+    pip3 install adafruit-circuitpython-bme280
```
Comment (kattni) [inline README.rst]: Same as above.

**Review by kattni — CHANGES_REQUESTED** [review]
Looks good! There's some copy/pasta that needs to be fixed. I've noted below. Thanks!

File: README.rst (hunk 1 above)
Comment (brentru) [reply README.rst]: Replaced, was focusing on making sure the license info was correct and missed this (added after)

**Review by kattni — APPROVED** [review]

**Pull request merged**